README.md

```diff
@@ -33,6 +33,26 @@ The software is provided under [Apache-2.0 license](LICENSE). Contributions to t
 
 This project contains code from other projects. The original license text is included in those source files. They must comply with our [license guide](https://os.mbed.com/docs/latest/reference/license.html)
 
+Folders containing files not under Apache 2.0 as listed above (each folder should contain own README file with license specified for its files). The original license text is included in those source files.
+
+- [cmsis](./cmsis) - MIT, BSD-3-Clause
```

Comment:
I _really_ don't like how much this changes what is suppose to be a simple README.